From 448df48c1610745cad734df210aa2f21f26c9ae4 Mon Sep 17 00:00:00 2001
From: Gunnar Roth <gunnar.roth@gmx.net>
Date: Fri, 22 Jan 2016 22:38:17 +0100
Subject: [PATCH] Fixing the SQLite3 build for WEC2013 again.

The new version broke the build again
-> fix it again.

Change-Id: I75761d134d97a2784f1de5076412aa814fdf9bcd

diff --git a/src/3rdparty/sqlite/sqlite3.c b/src/3rdparty/sqlite/sqlite3.c
index c0ab233..b397726 100644
--- a/src/3rdparty/sqlite/sqlite3.c
+++ b/src/3rdparty/sqlite/sqlite3.c
@@ -16867,6 +16867,13 @@ static void clearYMD_HMS_TZ(DateTime *p){
 #define HAVE_LOCALTIME_S 1
 #endif
 
+#if defined(_WIN32_WCE) && _WIN32_WCE < 0x800
+#undef HAVE_LOCALTIME_S
+struct tm *__cdecl localtime(const time_t *t);
+#elif defined(_WIN32_WCE) && _WIN32_WCE >= 0x800
+#define SQLITE_MSVC_LOCALTIME_API 1
+#endif
+
 #ifndef SQLITE_OMIT_LOCALTIME
 /*
 ** The following routine implements the rough equivalent of localtime_r()
-- 
2.5.0

